src/Bindings/Plugin.cpp

```diff
@@ -43,6 +43,16 @@ void cPlugin::Unload(void)
 
 
 
+void cPlugin::Reload(void)
+{
+	Unload();
+	Load();
+}
+
+
+
+
+
 AString cPlugin::GetLocalFolder(void) const
 {
 	return std::string("Plugins/") + m_FolderName;
```

Comment:
i know this is not your business but can you do
```suggestion
	return AString("Plugins") + cFile::GetPathSeparator() + m_FolderName;
```

Reply:
I'm just wondering why this didn't break in windows OS yet